### PR TITLE
Fix strike summary and familiar sheet conditions/effects, make animation smoother

### PR DESCRIPTION
**Pull request by CarlosFdez**

### CODE REVIEW

File: static/templates/actors/familiar-sheet.hbs [hunk 1]
```diff
@@ -210,32 +210,7 @@
...
-                                                <a class="item-control item-edit" title="{{localize "PF2E.EditItemTitle"}}"><i class="fas fa-edit"></i></a>
-                                                <a class="item-control item-delete" title="{{localize "PF2E.DeleteItemTitle"}}"><i class="fas fa-trash"></i></a>
-                                            </div>
-                                        {{/if}}
-                                        <div class="item-summary"></div>
-                                    </li>
-                                {{/if}}
-                            {{/if}}
-                        {{/each}}
+                        {{> systems/pf2e/templates/actors/partials/conditions.hbs conditions=document.itemTypes.effect}}
```
Comment (stwlam) [inline static/templates/actors/familiar-sheet.hbs:213]: Could we have a standard of quoting paths like this? The only thing that keeps it from getting mixed up with variables is it parsing like a path. I don't know that I want to trust handlebars to not fuck that up.

Comment (CarlosFdez) [reply static/templates/actors/familiar-sheet.hbs:213]: I suspect its actually a unique feature of {{> judging by this:
https://handlebarsjs.com/guide/partials.html#basic-partials

unless of course those are all variables in the global context somehow, which sounds pretty unsafe.

But sure, I'll update them and a bunch of other ones.

File: static/templates/actors/character/limited.hbs [hunk 2]
```diff
@@ -46,7 +46,7 @@
             <div class="tab actions actions-pane" data-group="primary" data-tab="effects">
                 <ol class="actions-list directory-list strikes-list">
                     <li class="item action-header stroke-header">
-                        {{> systems/pf2e/templates/actors/partials/images/header_stroke.hbs}}
+                        {{> "systems/pf2e/templates/actors/partials/images/header_stroke.hbs" }}
```
Comment (stwlam) [inline static/templates/actors/character/limited.hbs:49]:
```suggestion
                        {{> "systems/pf2e/templates/actors/partials/images/header_stroke.hbs"}}
```
:o)

Comment (CarlosFdez) [reply static/templates/actors/character/limited.hbs:49]: I see you don't like symmetry

Comment (stwlam) [reply static/templates/actors/character/limited.hbs]: I love it! the > is the first character inside the curly braces, and it's touching.

Comment (CarlosFdez) [reply static/templates/actors/character/limited.hbs]: done but I didn't test it

File: static/templates/actors/character/limited.hbs [hunk 3]
```diff
@@ -46,7 +46,7 @@
             <div class="tab actions actions-pane" data-group="primary" data-tab="effects">
                 <ol class="actions-list directory-list strikes-list">
                     <li class="item action-header stroke-header">
-                        {{> systems/pf2e/templates/actors/partials/images/header_stroke.hbs}}
+                        {{> "systems/pf2e/templates/actors/partials/images/header_stroke.hbs"}}
                         <h3 class="item-name">{{localize 'PF2E.ConditionsLabel'}}</h3>
```
Comment (stwlam) [inline static/templates/actors/character/limited.hbs:50]:
```suggestion
                        <h3 class="item-name">{{localize "PF2E.ConditionsLabel"}}</h3>
```

File: static/templates/actors/character/sidebar/armor-class.hbs [hunk 4]
```diff
@@ -84,5 +84,5 @@
     </div>
 </div>
 {{#with data.attributes.ac}}
-{{> systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs title="PF2E.ArmorClassLabel" }}
+    {{> "systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs" title="PF2E.ArmorClassLabel" }}
```
Comment (stwlam) [inline static/templates/actors/character/sidebar/armor-class.hbs:87]:
```suggestion
    {{> "systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs" title="PF2E.ArmorClassLabel"}}
```

File: static/templates/actors/character/sidebar/health.hbs [hunk 5]
```diff
@@ -46,5 +46,5 @@
     </div>
 </div>
 {{#with data.attributes.hp}}
-{{> systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs title="PF2E.HitPointsHeader" }}
+    {{> "systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs" title="PF2E.HitPointsHeader" }}
```
Comment (stwlam) [inline static/templates/actors/character/sidebar/health.hbs:49]:
```suggestion
    {{> "systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs" title="PF2E.HitPointsHeader"}}
```

File: static/templates/actors/character/sidebar/perception.hbs [hunk 6]
```diff
@@ -16,11 +16,11 @@
...
         <select class="perception-proficiency pf-rank adjust-stat-select{{#if rank.immutable}} readonly{{/if}}" data-property="system.attributes.perception.rank" data-rank="{{data.attributes.perception.rank}}" data-dtype="Number">
-            {{> systems/pf2e/templates/actors/character/partials/proficiencylevels-dropdown.hbs proflevel=data.attributes.perception.rank}}
+            {{> "systems/pf2e/templates/actors/character/partials/proficiencylevels-dropdown.hbs" proflevel=data.attributes.perception.rank}}
         </select>
     </div>
 
 </div>
 {{#with data.attributes.perception}}
-    {{> systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs title="PF2E.PerceptionHeader" }}
+    {{> "systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs" title="PF2E.PerceptionHeader" }}
```
Comment (stwlam) [inline static/templates/actors/character/sidebar/perception.hbs:25]:
```suggestion
    {{> "systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs" title="PF2E.PerceptionHeader"}}
```

File: static/templates/actors/character/sidebar/saves.hbs [hunk 7]
```diff
@@ -8,21 +8,21 @@
...
+                    {{> "systems/pf2e/templates/actors/character/icons/d20.hbs"}}
                 </a>
                 <h3>{{numberFormat save.totalModifier decimals=0 sign=true}}</h3>
             </div>
             <button type="button" class="modifiers_button hover" data-tooltip-content="#{{@root.options.id}}-{{save.slug}}-modifiers">
                 {{localize "PF2E.ModifiersTitle"}}
             </button>
             {{#with save}}
-                {{> systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs title=label }}
+                {{> "systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs" title=label }}
```
Comment (stwlam) [inline static/templates/actors/character/sidebar/saves.hbs:25]:
```suggestion
                {{> "systems/pf2e/templates/actors/character/partials/modifiers-tooltip.hbs" title=label}}
```

File: static/templates/actors/character/tabs/proficiencies.hbs [hunk 8]
```diff
@@ -10,14 +10,14 @@
                         <h4 class="skill-name rollable">{{localize skill.label}}</h4>
                         <div class="skill-prof button-group">
                             <select class="skill-proficiency pf-rank adjust-stat-select" data-property="system.skills.{{key}}.rank" data-rank="{{skill.rank}}" data-dtype="Number">
-                                {{> systems/pf2e/templates/actors/character/partials/proficiencylevels-dropdown.hbs proflevel=skill.rank }}
+                                {{> "systems/pf2e/templates/actors/character/partials/proficiencylevels-dropdown.hbs" proflevel=skill.rank }}
```
Comment (stwlam) [inline static/templates/actors/character/tabs/proficiencies.hbs:13]:
```suggestion
                                {{> "systems/pf2e/templates/actors/character/partials/proficiencylevels-dropdown.hbs" proflevel=skill.rank}}
```

File: static/templates/actors/character/limited.hbs (hunk 3 above)
Comment (CarlosFdez) [reply static/templates/actors/character/limited.hbs:50]: jesus! what have I ever done to you

**Pull request merged**